src/store/modules/keycodes/index.js

```diff
@@ -27,17 +31,58 @@ const state = {
   active: 'ANSI'
 };
 
-function isISO() {
-  return store.state.app.configuratorSettings.iso;
+function getOSKeyboardLayout() {
+  const fallbackOSKeyboardLayout = 'keymap_us';
+  const osKeyboardLayout =
+    store.state.app.configuratorSettings.osKeyboardLayout;
+  if (
+    isUndefined(osKeyboardLayout) ||
+    !Object.keys(keymapExtras).includes(osKeyboardLayout)
+  ) {
+    console.log(
+      `The stored OS keyboard layout value (${osKeyboardLayout}) is not a valid value! Falling back to '${fallbackOSKeyboardLayout}'.`
+    );
+    store.state.app.configuratorSettings.osKeyboardLayout =
```

Comment:
this isn't really a good idea. we should go through a commit to do this. I'd also use an actual getter on grabbing the state. now you know too much about the internals of store.state.app

Reply:
Solved in 8cb10a2